### PR TITLE
add inheritance for ERC677ReceiverInterface

### PR DESCRIPTION
**Pull request by cmalec**
add inheritance for ERC677ReceiverInterface

### CODE REVIEW

File: contracts/src/v0.8/KeeperRegistry.sol [hunk 1]
```diff
@@ -14,19 +14,21 @@ import "./interfaces/KeeperCompatibleInterface.sol";
...
-  KeeperRegistryExecutableInterface,
-  MigratableKeeperRegistryInterface
+TypeAndVersionInterface,
+ConfirmedOwner,
+KeeperBase,
+ReentrancyGuard,
+Pausable,
+KeeperRegistryExecutableInterface,
+MigratableKeeperRegistryInterface,
+ERC677ReceiverInterface
```
Comment (infiloop2) [inline contracts/src/v0.8/KeeperRegistry.sol:31]: needs prettier formatting

**Review by infiloop2 — COMMENTED** [review]
formatting required

File: contracts/src/v0.8/KeeperRegistry.sol [hunk 2]
```diff
@@ -63,7 +65,7 @@ contract KeeperRegistry is
   AggregatorV3Interface public immutable FAST_GAS_FEED;
 
   /**
-   * @notice versions:
+     * @notice versions:
```
Comment (infiloop2) [inline contracts/src/v0.8/KeeperRegistry.sol:68]: not sure how this passed prettier. Could you remove the space before merging

File: contracts/src/v0.8/KeeperRegistry.sol [hunk 3]
```diff
@@ -780,7 +782,8 @@ contract KeeperRegistry is
     uint256 premium = PPB_BASE + s_storage.paymentPremiumPPB;
     uint256 total = ((weiForGas * (1e9) * (premium)) / (linkEth)) + (uint256(s_storage.flatFeeMicroLink) * (1e12));
     if (total > LINK_TOTAL_SUPPLY) revert PaymentGreaterThanAllLINK();
-    return uint96(total); // LINK_TOTAL_SUPPLY < UINT96_MAX
+    return uint96(total);
+    // LINK_TOTAL_SUPPLY < UINT96_MAX
```
Comment (infiloop2) [inline contracts/src/v0.8/KeeperRegistry.sol:786]: same line was better imo

**Review by infiloop2 — APPROVED** [review]

File: contracts/src/v0.8/KeeperRegistry.sol (hunk 2 above)
Comment (cmalec) [reply contracts/src/v0.8/KeeperRegistry.sol:68]: ill change both these in the next one as ill have to merge in the base branch

**Pull request merged**